templates/BodyParamSnippet.stencil

```diff
@@ -1,3 +1,3 @@
-{% if op.params.body.strategy == "plain" or op.params.body.strategy == "unixTime"  %}
-    {{ op.params.body.param.name }}: {{ op.params.body.param.type }}{{ op.params.body.param.defaultValue }},
+{% if op.params.body.strategy == "plain" or op.params.body.strategy == "unixTime" or op.params.body.strategy == "plainNullable" or op.params.body.strategy == "byteArray" %}
```

Comment:
Is there a strategy for which this block doesn't apply? if so, it would probably be easier to change the logic to != for that.

Reply:
Great point.

Reply:
Try changing to `!=` but stencil can't resolve `{{ op.params.body.param.name }} ` and `{{ op.params.body.param.type }}` for some reason. So keep the `==` logic instead